lib/dynamics_crm/xml/attributes.rb

```diff
@@ -33,6 +33,8 @@ def get_type(key, value)
             type = "FetchExpression"
           when Money
             type = "Money"
+          when DynamicsCRM::Metadata::Double
```

Comment:
Why is a separate class type necessary?  The rest of the primitives are handled without a custom class.